rpm_spec/gui-agent.spec.in

```diff
@@ -161,6 +164,38 @@ echo autospawn=no >> /etc/pulse/client.conf
 rm -rf $RPM_BUILD_ROOT
 rm -f %{name}-%{version}
 
+%package selinux
+Summary: SELinux policy for the Qubes GUI Agent
+License: GPLv2+
+
+BuildRequires: selinux-policy
+BuildRequires: selinux-policy-devel
+BuildRequires: qubes-core-agent-selinux
```

Comment:
This one (and the other BR below) is problematic. This repo is built for dom0 too, and you don't have core-agent-linux in dom0 repository. The cause is different, but it is similar issue to https://github.com/QubesOS/qubes-core-qubesdb/pull/42#issuecomment-1187684397 and if possible, such BR should be avoided.

Without `qubes-core-agent-selinux`, `qubes-gui-agent-selinux` doesn't make much sense either, so an alternative could be excluding -selinux subpackage build for dom0, but it sounds more involved because it would require splitting -selinux subpackage into separate spec (we intentionally don't have rpm macro telling whether the build is for dom0 or not - as that could easily get you different content in packages with otherwise identical name, version etc).